frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```diff
@@ -196,6 +196,8 @@
   "Select a Project to view the list of Pipelines": "Select a Project to view the list of Pipelines",
   "Create Pipeline": "Create Pipeline",
   "by name": "by name",
+  "{{resource}} results": "{{resource}} results",
+  "No {{resource}} results available due to failure": "No {{resource}} results available due to failure",
```

Comment:
Hmmm... thought we agreed on `resourceName` to help with the translators...

Reply:
Ok, I'll make it `resourceName` for all the related ones.

Reply:
You changed the usages but not the definition... you still need to update the json file.